Sources/SessionTools/KeychainStorage/KeychainStorageContainer.swift

```diff
@@ -8,14 +8,36 @@
 import Foundation
 import KeychainAccess
 
+/// The lifecycle of the keychain item added
+public enum KeychainLifecycle: Equatable {
+    /// Default case - it's up to you and the OS rules to define when this session is deleted
+    case unmanaged
+    
+    /// The session will only live in the current installation - a delete and reinstall will make this item unreachable.
```

Comment:
Let's reword this slightly since technically, a delete and re-install will only make this item unreachable if a different `identifier` is used. Maybe something about how the session is entangled to the identifier so using the same identifier will allow the session to be restored, but using a different identifier will cause a new session to be started/used.